src/other_functions.jl

```diff
@@ -178,3 +179,70 @@ polynomials. For more details, see [`Base.isapprox`](@ref).
 function isapprox{T<:AbstractSeries,S<:AbstractSeries}(x::T, y::S; rtol::Real=rtoldefault(x,y), atol::Real=0, nans::Bool=false)
     x == y || (isfinite(x) && isfinite(y) && norm(x-y,1) <= atol + rtol*max(norm(x,1), norm(y,1))) || (nans && isnan(x) && isnan(y))
 end
+
+
+#taylor_expand function for Taylor1
+doc"""
+    taylor_expand(f,x0;order)
+
+Makes a Taylor expansion of the function `f` around the point `x0`. If x0 is a scalar,
+a `Taylor1` expansion will be done. If `x0` is a vector, a `TaylorN` expansion will be
+computed, changing the number of `TaylorN` variables
+according to the dimension of `f`.
+"""
+function taylor_expand(f::Function; order::Int64=15)
```

Comment:
Note that this will return a `Taylor1{Float64}` expansion. My point focuses in the `Float64` part.

Reply:
Do you think `taylor_expand(T::Type,f::Function;order::Int64)` is a good idea? In this case,  should the same be done when passing an explicit `x0` to `taylor_expand`?